src/Projection/User/UserProjector.php

```diff
@@ -105,4 +106,28 @@ public function onTodoWasMarkedAsDone(TodoWasMarkedAsDone $event)
 
         $stmt->execute();
     }
+
+    /**
+     * @param TodoWasReopened $event
+     * @throws \RuntimeException if data of the the assigned user can not be found
+     */
+    public function onTodoWasReopened(TodoWasReopened $event)
+    {
+        $user = $this->userFinder->findUserOfTodo($event->todoId()->toString());
```

Comment:
most things were a copy of the TodoWasMarkedAsDone behaviour, that's my excuse here... ;-p

> It is task of the domain to protect invariants

sounds sane